.github/ISSUE_TEMPLATE/1_Bug_report.md

```diff
@@ -11,12 +11,15 @@ labels: Bug
 <!-- A clear and concise description of the problem. -->
 
 **How to reproduce**
-<!-- Code and/or config needed to reproduce the problem. If it's a complex bug,
-     create a "bug reproducer" as explained in:
+<!-- \!/ This is the most important part of the report. \!/
+     Without a way to easily reproduce your issue, there is little chance we will be able to help you and work on a fix.
+     Please, take the time to show us some code and/or config that is needed to reproduce the problem.
```

Comment:
that is needed *for others* to reproduce

Reply:
done